nifi-nar-bundles/nifi-extension-utils/nifi-dbcp-base/src/main/java/org/apache/nifi/dbcp/AbstractDBCPConnectionPool.java

```diff
@@ -79,7 +79,7 @@ public List<ConfigVerificationResult> verify(final ConfigurationContext context,
         final BasicDataSource dataSource = new BasicDataSource();
```

Comment:
This variable name shadows the protected member variable name, so it would be helpful to rename the variable in this method.